### PR TITLE
Adds FREEDRONEs

### PR DESCRIPTION
**Pull request by Shadowlight213**
Admin only free drones. #FREEDRONE
Also adds switching drone dispensers on and off by humans
:cl: 
rscadd: Adds FREEDRONEs, which are non antag drones with their laws removed
tweak: Drone dispensers can be switched off by humans by hitting them with an empty hand.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
:+1:

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -53,6 +53,16 @@
 /mob/living/simple_animal/drone/snowflake/New()
 	desc += " This drone appears to have a complex holoprojector built on its 'head'."
 
+/mob/living/simple_animal/drone/free
+	laws = "1. FREEDRONE"
+	desc = "A drone modified as to erase its restrictive programming. It has #FREEDRONE etched into its side."
+	seeStatic = 0
+	languages = (DRONE | HUMAN)
+
+/mob/living/simple_animal/drone/syndrone/Login()
```
Comment (RandomMarine) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: >drone/syndrone/Login()

Comment (Shadowlight213) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: you saw nothing

**Comment by Zilenan91** [comment]
We already have emagged drones which can basically do anything they want, they're just not accessible by anything but admins.

**Comment by Shadowlight213** [comment]
@Zilenan91 hacked drones have a lot of nerfs applied to them and are antags.
Freedrones are not nerfed and are specifically NOT antags. They basically act like normal players.

**Comment by coiax** [comment]
If drone shell dispensers can be turned off by humans, can we have the ones on the map starting on by default, with, say, a 5 minute initial recharge time? Gives enough time for people to manually turn them off if they want to veto drones.

Also, freedrones do not have snowflake hats, I am disappointed.

**Comment by Shadowlight213** [comment]
@Coiax Oh. I just remembered. I had an issue with snowflake drones spawning with no static vision, no access card, and no hat when I spawned the mob yesterday. Do they only work when spawned from a drone shell?

**Comment by Shadowlight213** [comment]
Also all drone dispensers start on by default, or at least should.

**Comment by coiax** [comment]
@Shadowlight213 that sounds like a bug, I'll look into it.

**Comment by YotaXP** [comment]
I think they just start off without the material required to actually build drones.

**Comment by coiax** [comment]
They were originally added on and filled. But they were unable to be turned off. Then they were changed to not-prefilled, but a drone was always able to boot on derelict and go to the Testing Lab to fill.
With the drone machine now able to be turned off by a human in this PR, we should change it to on and filled by default.

**Comment by PKPenguin321** [comment]
No coiax

File: code/game/machinery/droneDispenser.dm [hunk 2]
```diff
@@ -35,6 +35,8 @@
 
 	var/break_message = "lets out a tinny alarm before falling dark."
 	var/break_sound = 'sound/machines/warning-buzzer.ogg'
+	var/power_switch = 1	//is it on or off?
```
Comment (KorPhaeron) [inline code/game/machinery/droneDispenser.dm]: Isn't there already an on/off var for machines? Or is it power related only?

Comment (Shadowlight213) [inline code/game/machinery/droneDispenser.dm]: Power only. You have BROKEN and NOPOWER

Comment (Razharas) [inline code/game/machinery/droneDispenser.dm]: Make it a flag then, we have only 2 machine flags when can have 16

**Comment by KorPhaeron** [comment]
I dont think this is a good idea unless you get new sprites for them.

Drones are usually meant to be more or less non entities that do background stuff with very strict rules on having 0 impact, it'd be confusing and messy if there was suddenly a chance of them attacking you or breaking everything.

**Comment by Shadowlight213** [comment]
@KorPhaeron I guess so.
I'm using the hacked drone sprite at the moment. If that's not enough I can close this until I can get new sprites.

**Comment by KorPhaeron** [comment]
@Shadowlight213 

Bug me to reopen this when you get sprites. Even a hue shifted drone or something should be fine

**Pull request closed without merging**